yoti-sdk/src/main/java/com/yoti/mobile/android/sdk/kernelSDK/KernelSDKIntentService.java

```diff
@@ -140,6 +139,11 @@ private void handleActionStartScenario(String useCaseId, ResultReceiver resultRe
             intent.setData(uri);
         }
 
+        Intent wakeupIntent = new Intent(this, ReceiverActivity.class);
```

Comment:
I am not sure going through a transparent is the best/only solution here. Even if it works it looks like a workaround
Why not requiring the integrator to return the name of the target component to needs to be woken up? 
Also this is an `IntentService` has background limitation since Android 8.0 and it is recommended to use a `JobIntentService` maybe this is related to the problem you are trying to fix.

Reply:
It is kind of a workaround but for a very good reason,  with this implementation integrators won't have to make any modifications in their apps.

In this PR we are focusing on dealing with the new restriction in Android Q to open activities from the background.

Reply:
Ok and I suspect the change to `JobIntentService` will be taken care of in a different ticket?

Reply:
Yes, it is a separate issue. :-)